awx/main/tasks.py

```diff
@@ -905,6 +905,46 @@ def build_params_process_isolation(self, instance, private_data_dir, cwd):
                 process_isolation_params['process_isolation_ro_paths'].append(instance.ansible_virtualenv_path)
         return process_isolation_params
 
+    def build_params_resource_profiling(self, instance, private_data_dir):
+        resource_profiling_params = {}
+        if self.should_use_resource_profiling(instance):
+            base_cgroup = getattr(settings, 'AWX_RESOURCE_PROFILING_BASE_CGROUP', 'ansible-runner')
```

Comment:
If you set these in `defaults.py`, you can get rid of the `getattr` calls here and just do `settings.XYZ`.